packages/suite/src/views/wallet/transactions/TransactionList/TransactionsGroup/PendingGroupHeader.tsx

```diff
@@ -0,0 +1,14 @@
+import { Translation } from 'src/components/suite';
+import { ColPending, HeaderWrapper } from './CommonComponents';
+
+type PendingGroupHeaderProps = { txsCount: number };
+
+export const PendingGroupHeader = ({ txsCount }: PendingGroupHeaderProps) => {
```

Comment:
ultra-nit: inline function

Reply:
I'd keep this. We may need to add some logic later.